### DISCUSSION TITLE
Hide Install banner

### DISCUSSION THREAD

**Discussion by brlarini** [rustdesk/rustdesk#5039] [Q&A]
On Windows, is there any filename option to hide/remove the Install banner on the main screen?
Thanks!

**Comment by SergeyMy** [comment]
for what purpose?

**Reply by brlarini** [reply]
Just to keep clueless users from installing it when the standalone/portable mode is enough.
There's no problem if they manage to install it (in some cases it could be useful), I just wanted to *avoid* them doing so by themselves.

Thanks!

**Reply by SergeyMy** [reply]
but it's true :) give them the red button and they will definitely press it
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/f6f4d345-4835-4367-af41-2b8f6bd2d8a7)
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/d26d8e84-cf28-42ff-bd79-f7f206d4616d)
